app/models/manageiq/providers/openstack/infra_manager/host/operations.rb

```diff
@@ -0,0 +1,24 @@
+module ManageIQ::Providers::Openstack::InfraManager::Host::Operations
+  def self.included(base)
+    base.send :include, InstanceMethods
+    base.extend ClassMethods
+  end
+
+  module InstanceMethods
+    def ironic_fog_node
+      fog_baremetal = ext_management_system.openstack_handle.baremetal_service
+      fog_baremetal.nodes.find_by_uuid(uid_ems)
```

Comment:
Gross.  @Ladas, is that a `fog_openstack` bug?

Can be fixed later after an update to `fog_openstack`.